Assets/Mirror/Runtime/NetworkReader.cs

```diff
@@ -39,6 +38,20 @@ public NetworkReader(ArraySegment<byte> segment)
             buffer = segment;
         }
 
+        // set buffer methods mirror constructor for ReaderPool
+        public void SetBuffer(byte[] bytes)
```

Comment:
maybe make them internal so only our pool can use them?
might be a bit confusing / risky to expose them to everyone

Reply:
Done